### DISCUSSION TITLE
Hacker News should have links to the site

### DISCUSSION THREAD

**Discussion by theklr** [dailydotdev/daily#579] [Feature Requests]
Would be nice for sites with discussions that have links both have the option of going to the direct link and/or the related topic on hacker news.

**Comment by idoshamun** [comment]
Hi @theklr,

Thanks for the suggestion. We'll look into the technical aspect of this request :)

**Reply by rdrey** [reply]
came here to request the same thing. Links from sites with discussions on them like HN should also link to the discussion
